packages/automerge-repo/src/storage/chunkTypeFromKey.ts

```diff
@@ -0,0 +1,22 @@
+import { StorageKey } from "./types.js"
+import { ChunkType } from "./types.js"
+
+/**
+ * Keys for storing Automerge documents are of the form:
+ * ```ts
+ * [documentId, "snapshot", hash]  // OR
+ * [documentId, "incremental", hash]
+ * ```
+ * This function returns the chunk type ("snapshot" or "incremental") if the key is in one of these
+ * forms.
+ */
+export function chunkTypeFromKey(key: StorageKey): ChunkType | null {
```

Comment:
i'm not sure there's a good reason to differentiate these anymore. what do you think?

Reply:
It's true that we're not doing anything with the information except to ensure that it's one or the other, but I think the additional clarity doesn't hurt anything & could be helpful for debugging etc.